ext/ffi_yajl/ext/parser/parser.c

```diff
@@ -23,6 +24,12 @@ void set_value(CTX *ctx, VALUE val) {
       rb_ary_push(last, val);
       break;
     case T_HASH:
+      if ( ((CTX *)ctx)->uniqueKeyChecking ) {
```

Comment:
you are correct, copypasta'd that from another function where its declared as a void *ctx.